### PR TITLE
release 1.19.0

### PR DESCRIPTION
**Pull request by philbrookes**
@aidenkeating @KevFan can you check the updates you require a present here, please

### CODE REVIEW

File: deploy/olm-catalog/integreatly-operator/integreatly-operator-1.19.0/integreatly-operator.v1.19.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -35,29 +35,24 @@ metadata:
...
+    support: RHMI
+  name: integreatly-operator.v1.19.0
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - kind: RHMI
-      description: Represents an installation of RHMI.
-      displayName: RHMI Installation
```
Comment (KevFan) [inline deploy/olm-catalog/integreatly-operator/integreatly-operator-1.19.0/integreatly-operator.v1.19.0.clusterserviceversion.yaml]: Just testing an olm install and looks like we need at least `displayName` in `spec.customresourcedefinitions.owned` for the InstallPlan & Subscription to resolve. 

I tested generating a new csv based off this one, and it looks like these fields no longer get brought over from the previous csv using operator-sdk 0.15.1 :confused: :man_shrugging:

File: deploy/olm-catalog/integreatly-operator/integreatly-operator-1.19.0/integreatly-operator.v1.19.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -258,14 +253,16 @@ spec:
                     fieldRef:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
-                  value: integreatly-operator
+                  value: rhmi-operator
                 - name: USE_CLUSTER_STORAGE
                   value: "true"
-                image: quay.io/integreatly/integreatly-operator:v1.18.0
+                - name: ALERTING_EMAIL_ADDRESS
+                  value: "integreatly-notifications@redhat.com"
```
Comment (aidenkeating) [inline deploy/olm-catalog/integreatly-operator/integreatly-operator-1.19.0/integreatly-operator.v1.19.0.clusterserviceversion.yaml]: @philbrookes Looks good.

To confirm, his would only be set in addon environments from stage, is that correct? This won't be set in any dev environments? If that's the case, we're all good

Comment (philbrookes) [reply deploy/olm-catalog/integreatly-operator/integreatly-operator-1.19.0/integreatly-operator.v1.19.0.clusterserviceversion.yaml]: ah, I see then I should remove it from here.

File: deploy/olm-catalog/integreatly-operator/integreatly-operator-1.19.0/integreatly-operator.v1.19.0.clusterserviceversion.yaml (hunk 1 above)
Comment (philbrookes) [reply deploy/olm-catalog/integreatly-operator/integreatly-operator-1.19.0/integreatly-operator.v1.19.0.clusterserviceversion.yaml]: I've re-added these now

**Comment by KevFan** [comment]
This looks good to me anyway so going to approve

/lgtm
/approve

**Pull request merged**